scripts/ghidra/PatchestryDecompileFunctions.java

```diff
@@ -1631,6 +1830,14 @@ private boolean fixupOperations(
 								}
 							}
 							break;
+						case PcodeOp.CALL:
+							// Rewrite call argument if there is type mismatch and can't be
+							// handled during AST generation
+							rewriteCallArgument(high_function, op);
+							// Rewrite return type if the function retuns void but pcode op has 
+							// valid output varnode.
+							rewriteVoidReturnType(high_function, op);
```

Comment:
To make sure I fully understand: function return type rewriting is necessary when the function produced *is*, but based on what was in the pcode, should not be void?